src/library/scala/collection/Iterable.scala

```diff
@@ -26,6 +26,10 @@ trait Iterable[+A] extends IterableOnce[A] with IterableOps[A, Iterable, Iterabl
   protected def fromSpecificIterable(coll: Iterable[A @uncheckedVariance]): IterableCC[A] @uncheckedVariance = iterableFactory.from(coll)
   protected def newSpecificBuilder: Builder[A, IterableCC[A]] @uncheckedVariance = iterableFactory.newBuilder[A]
 
+  @deprecated("Use newSpecificBuilder instead", "2.13.0")
+  @deprecatedOverriding("Override newSpecificBuilder instead", "2.13.0")
```

Comment:
Will overriding `newBuilder` be sufficient to preserve the old behavior? If not, this `deprecatedOverriding` is not strong enough, and `newBuilder` must be `final` instead.